app/assets/javascripts/documents.js.coffee

```diff
@@ -19,6 +19,24 @@
 # TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.~
 
 $ ->
+
+  setDocumentTableInterval = null
+
+  $(document).on 'click', '.report-request', ->
+    setDocumentTableInterval = setInterval refreshDocumentTable, 5000
+
+  clearDocumentTableInterval = (complete) ->
+    if complete == true
+      clearInterval setDocumentTableInterval
+
+  refreshDocumentTable = ->
+    $('#documents_table').bootstrapTable('refresh', {silent: "true"})
+    complete = true
+    $('span').each ->
```

Comment:
This could be cleaned up a little by using something like
```
if $('span.processing').length
  complete = false
```
In place of the .each iteration

Reply:
Done.